### ISSUE TITLE
Can i use Llama 2 instead of chatGpt 4, is it even possible here ?

### ISSUE THREAD

**Issue by phpmind** [chapyter/chapyter#27]
Can i use Llama 2 instead of chatGpt 4, is it even possible here?

its a silly question but I have to ask. I do not know much. 

Thanks

**Comment by lolipopshock** [comment]
Yes it is possible and adding this is function on our roadmap :)

**Comment by alanpaulkwan** [comment]
One LLM I might advocate for, in particular, is Claude since it can handle much longer contexts than either Llama or ChatGPT can.

**Comment by phpmind** [comment]
Does our code support Claude ?  If it does community will be delighted !!
